src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarExtraLinks.tsx

```diff
@@ -1,36 +1,79 @@
 import { Box, Sans, Spacer } from "@artsy/palette"
-import React from "react"
+import { ContextConsumer } from "Artsy/Router"
+import React, { SFC } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
 
 import { ArtworkSidebarExtraLinks_artwork } from "__generated__/ArtworkSidebarExtraLinks_artwork.graphql"
+import { Mediator } from "Artsy/SystemContext"
+import { data as sd } from "sharify"
 
 export interface ArtworkSidebarExtraLinksProps {
   artwork: ArtworkSidebarExtraLinks_artwork
 }
 
+interface ArtworkSidebarExtraLinksContainerProps
+  extends ArtworkSidebarExtraLinksProps {
+  mediator: Mediator
+}
+
 const Container = ({ children }) => (
   <Sans size="2" color="black60">
     {children}
   </Sans>
 )
 
-export class ArtworkSidebarExtraLinks extends React.Component<
-  ArtworkSidebarExtraLinksProps
+const Link = styled.a`
+  text-decoration: underline;
+`
+
+class ArtworkSidebarExtraLinksContainer extends React.Component<
+  ArtworkSidebarExtraLinksContainerProps
 > {
+  onClickAuctionFAQ() {
+    this.props.mediator &&
```

Comment:
#minor since this is a required prop I think this first part (`this.props.mediator &&`) is always true.